dev/provider_packages/README.md

```diff
@@ -171,3 +171,124 @@ the version in PyPI does not contain the leading 0s in version name - therefore
 also do not container the leading 0s.
 
 * You can install the .whl packages with `pip install <PACKAGE_FILE>`
+
```

Comment:
I've added document about manual procedure for testing if the provider/backport packges are importable. That saves a lot of time for testing of those scripts because you can initialize the whole container with all the deps once and tests the scripts inside.